### PR TITLE
Julian Aiup Topic0 - Exercises 1.The Basics of HTML

### PR DESCRIPTION
**Pull request by jaiup**

### CODE REVIEW

**Review by afiorenza — COMMENTED** [review]
Well done!

File: 00-HTML-CSS-basics/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        <link rel="stylesheet" href="css/style.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <!--[if lte IE 9]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+        <![endif]-->
+
+        <!-- Add your site or application content here -->
+        <header>
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/index.html]: It seems to have some indentation issues

File: 00-HTML-CSS-basics/sign-up-form.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Sign Up Form</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <form>
+      First name: <input type="text" name="first-name" id="first-name" value="" required="" />
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/sign-up-form.html]: It is not mandatory but is a good practice (also for accessibility) to have a label for each input.

**Review by afiorenza — COMMENTED** [review]
Good job! just a few comments

File: 00-HTML-CSS-basics/index.html (hunk 1 above)
Comment (jaiup) [reply 00-HTML-CSS-basics/index.html]: Yes, I’ve downloaded the file and didn’t notice that, sorry

**Review by afiorenza — APPROVED** [review]
Well done! ready for merge

**Comment by jaiup** [comment]
@afiorenza finished last exercise

**Pull request merged**